GDJS/Runtime/pixi-renderers/pixi.js

```diff
@@ -1160,4 +1160,4 @@ void main(void)\r
 // Disabled to avoid warnings in Chrome (see https://github.com/4ian/GDevelop/pull/3947)
-// //# sourceMappingURL=pixi-legacy.min.js.map
+//# sourceMappingURL=pixi-legacy.min.js.map
```

Comment:
Can you confirm this does not create a warning in Electron/when running a preview?

Reply:
Oh, I saw this issue in the comments. I uncommented it for debugging. Let me restore

Reply:
done